packages/flutter/test/material/menu_anchor_test.dart

```diff
@@ -3960,6 +3960,65 @@ void main() {
         tester.getRect(find.byKey(contentKey)).left + horizontalOffset,
       );
     });
+
+    group('The menu is attached to the bottom of the MenuAnchor content', () {
```

Comment:
<img width="627" alt="image" src="https://github.com/user-attachments/assets/3be80327-9c2e-4772-a4a5-b444b3a7cb3b">

The group name may be a bit too specific considering the other groups in this testing repo. This may be a matter of making the other groups more specific.